docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,10 +367,13 @@ which is a CDI bean and can be ``@Inject``ed.
 
 In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+NOTE: This configuration is only relevant for _Jakarta Transactions_ transactions.
 
-Quarkus allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store.<property>` properties, where <property> can be:
+However, in cloud-native apps, it is important to assess JTA transactions after careful evaluation.
+The Narayana framework, which manages these transactions, requires stable storage, a unique reusable node identifier, and a steady IP address to work correctly.
+While the JDBC object store handles one of these needs, users must still plan how to meet the other two requirements.
 
+Quarkus then allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store._<property>_` properties, where _<property>_ can be:
 
 * `type` (_string_): Configure this property to `jdbc` to enable usage of a Quarkus JDBC datasource for transaction logging.
```

Comment:
I assume "transaction logging" is referring to transaction logs as opposed to log messages. To be consistent with the rest of the document I would write `storing transaction logs` instead of "for transaction logging".